packages/fx-core/src/plugins/resource/bot/v2/common.ts

```diff
@@ -31,17 +35,14 @@ export function getTemplateInfos(ctx: Context, inputs: Inputs): CodeTemplateInfo
 }
 
 export function decideTemplateScenarios(ctx: Context, inputs: Inputs): Set<string> {
-  const isM365 = ctx.projectSetting?.isM365;
```

Comment:
Any background about deprecating the use of `isM365` in projectSettings?

Reply:
We try to reduce the flags for deciding the template, so that the `decideTemplateScenarios` function would be simpler.